docs/GlutenUsage.md

```diff
@@ -4,26 +4,9 @@ Based on the different environment, there are some parameters can be set via -D
-| arrow_root                                            | When build_arrow set to False, arrow_root will be enabled to find the location of your existing arrow library.                                                                      | /PATH_TO_GLUTEN/ep/build-arrow/build/arrow_install |
-| build_protobuf                                        | Build Protobuf from Source. If set to False, default library path will be used to find protobuf library.                                                                            | ON                                                 |
-| build_folly                                           | Build folly from Source.                                                                                                                                                            | ON                                                 |
-| enable_ep_cache                                       | Whether to cache the source folder for some critical external projects (e.g. Arrow, Velox) to spped-up build process. Note, this will not be invalidated even you use 'mvn clean'.  | OFF                                                |
 | backends-velox                                        | Add -Pbackends-velox in maven command to compile the JVM part of Velox backend.                                                                                                     | false                                              |
+| backends-gazelle                                        | Add -Pbackends-gazelle in maven command to compile the JVM part of Gazelle backend.                                                                                                     | false                                              |
 | backends-clickhouse                                   | Add -Pbackends-clickhouse in maven command to compile the JVM part of ClickHouse backend.                                                                                           | false                                              |
-| build_velox_backend                                   | Enable or Disable building the CPP part of Velox backend.                                                                                                                           | OFF                                                |
-| build_velox_from_source                               | Enable or Disable building Velox from a specific velox github repository. A default installed path will be in velox_home.                                                           | OFF                                                |
-| velox_home (only valid when _backend is ON)           | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                              | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep      |
```

Comment:
maybe a mistake delete?

Reply:
> maybe a mistake delete?

we add parameters to shell script and mvn will not use it.

Reply:
Can you document shell script argument?

Reply:
fixed. Added comments for all script argument.